docs/core-concepts/index.md

```diff
@@ -0,0 +1,36 @@
+# Core Concepts
+
+These are some of the core concepts that you should be familiar with before going over the [Getting Started Guide](get-started/index.md).
+
+## GitOps
+
+GitOps is a way of managing your infrastructure and applications so that whole system is described declaratively and version controlled(most likely in a Git repository), and having an automated process that ensure that the deployed environment matches the state specified in a repository.
```

Comment:
```suggestion
GitOps is a way of managing your infrastructure and applications so that whole system is described declaratively and version controlled (most likely in a Git repository), and having an automated process that ensure that the deployed environment matches the state specified in a repository.
```

In addition,

> [..] process that ensure that the deployed [..]

should either be

> [..] process that ensures that the deployed [..]

or

> [..] process to ensure that the deployed [..]